codify/codify.go

```diff
@@ -138,7 +138,7 @@ func alias(generated, defaultalias string) string {
 }
 
 func varName(name string) string {
-	reg, _ := regexp.Compile("[^a-zA-Z0-9 ]+")
+	reg, _ := regexp.Compile("[^a-zA-Z0-9 \\-]+")
```

Comment:
Added the - to the allow list. There is a space before this, not sure if space before it is necessary or wanted. I left it for the moment, we can patch if necessary.